### ISSUE TITLE
Show hashsums of restorable files

### ISSUE THREAD

**Issue by nettnikl** [emuell/restic-browser#63]
Hi,

i'm sometimes interested in checking the content of a file before starting a lengthy extraction/download process. Is there a way to show hashsums of restorable files in the list similar to Mtime, Atime,...? I'm not sure, but i though the metadata of restic contains some kind of standard hashsum for each file anyways.

**Comment by emuell** [comment]
The `restic ls --json` command unfortunately does not include hashes:

See https://github.com/restic/restic/blob/master/doc/075_scripting.rst#node

**Comment by nettnikl** [comment]
Yes, evem worse: i've tried around yesterday, and i was not able to find a way to easily extract the hashes at all - but maybe there is one i don't know. I'll probably go for a solution where i just pipe it into the sha cli.
